### DISCUSSION TITLE
Sidebar not scrolling in Astro starlight docs

### DISCUSSION THREAD

**Discussion by temiye18** [mearashadowfax/ScrewFast#213] [Q&A] [answered]
I noticed the scroll bar is not scrolling when I swipe up and down my mouse pad or use an external mouse's scroll wheel. It only works when the scroll bar is manually clicked on and dragged. What could be the problem? I have a stand-alone astro starlight project and I works there.

<img width="374" alt="Screenshot 2024-09-25 094427" src="https://github.com/user-attachments/assets/7a1085c3-adc5-4be4-8f45-bd53b655540a">

**Comment by mearashadowfax** [comment, marked as answer]
Hi @temiye18! I checked, and the issue is due to the Lenis smooth scroll library. You can disable it by removing the following code:
```astro
<script>
    import "@scripts/lenisSmoothScroll.js";
  </script>
```
 from `src/components/ui/starlight/Head.astro`.

**Reply by temiye18** [reply]
Thanks a lot. It has worked
